README.md

```diff
@@ -77,6 +77,24 @@ The hot-reload is manual because you'll need to press 'r' in the terminal to hot
 
 By default, hover uses the file `lib/main_desktop.dart` as entrypoint. You may specify a different endpoint by using the `--target` flag.
 
+You can also use the configuration in the pubspec.yaml (this is the default configuration):
+```yaml
+name: ...
+description: ...
+version: ...
+
+dependencies: ...
+
+flutter: ...
+
+go-flutter:
+    target: lib/main_desktop.dart
+    branch: "@master"
+    cache-path: "/home/YOURUSERNAME/.cache/"
+    opengl: "3.3"
```

Comment:
Usually people don't really know what is this flag about.
I suggest having it commented to `opengl: "none"` with a comment saying "If you having trouble the OpenGL driver (especially on virtual machine), un-commenting this line" 

Some context for you @jld3103: https://github.com/go-flutter-desktop/go-flutter/issues/272